README.cn.md

```diff
@@ -0,0 +1,342 @@
+```
+
+最后一个您可能遇到的错误是：
+
+```
+ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
+```
+
+这不是身份验证问题，而是告诉您数据库服务器没有运行。 如下启动数据库服务器：
+
+```sh
+sudo service mysql start
+```
+
+### Unknown authentication method错误
+
+PHP 无法再与最新版本的 MySQL 默认配置中的数据库通信。 如果您尝试运行配置脚本并收到以下消息，则表示您正在使用这个默认配置。
+
+```
+Database Error #2054: The server requested authentication method unknown to the client.
+```
+
+您有两个选择，最简单的是卸载 MySQL 并安装 MariaDB。 以下是来自 MariaDB 项目的官方指南：
+
+<https://mariadb.com/resources/blog/how-to-migrate-from-mysql-to-mariadb-on-linux-in-five-steps/>
+
+或者，按照以下步骤操作：
+1. 以 root 用户身份编辑以下文件：`/etc/mysql/mysql.conf.d/mysqld.cnf`
+2. 在 `[mysqld]` 行下，添加以下内容： `default-  authentication-plugin=mysql_native_password`
+3. 重启数据库：`sudo service mysql restart`
+4. 检查数据库用户的身份验证方法：
+
+    ```sql
+    mysql> select Host,User, plugin from mysql.user where mysql.user.User = 'dvwa';
+    +-----------+------------------+-----------------------+
+    | Host      | User             | plugin                |
+    +-----------+------------------+-----------------------+
+    | localhost | dvwa             | caching_sha2_password |
+    +-----------+------------------+-----------------------+
+    1 rows in set (0.00 sec)
+    ```
+5. 你可能会看到 `caching_sha2_password`。 如果是这样，请运行以下命令：
+
+    ```sql
+    mysql> ALTER USER dvwa@localhost IDENTIFIED WITH mysql_native_password BY 'p@ssw0rd';
+    ```
+6. 重新运行检查，您现在应该看到`mysql_native_password`。
+    ```sql
+    mysql> select Host,User, plugin from mysql.user where mysql.user.User = 'dvwa';
+    +-----------+------+-----------------------+
+    | Host      | User | plugin                |
+    +-----------+------+-----------------------+
+    | localhost | dvwa | mysql_native_password |
+    +-----------+------+-----------------------+
+    1 row in set (0.00 sec)
+    ```
+
+经过以上步骤，设置过程现在应该可以正常工作了。
+
+如果您想了解更多信息，请参阅以下页面：<https://www.php.net/manual/en/mysqli.requirements.php>。
+
+### 数据库错误 Error #2002: No such file or directory.
+
+数据库服务器没有运行。 在基于 Debian 的发行版中，这可以通过以下方式完成：
+
+```sh
+sudo service mysql start
+```
+
+### "MySQL server has gone away" 和 "Packets out of order" 错误
+
+出现这些错误的原因有多种，但最有可能的原因是您运行的数据库服务器版本与 PHP 版本不兼容。
+
+最常见的是，你运行了最新版本的 MySQL 与 PHP 搭配部署，而这两者并不十分兼容。最好的建议是，放弃 MySQL 并安装 MariaDB，因为这（译注：使用最新版MySQL）不是我们可以支持的。
+
+有关更多信息，请参阅：
+
+<https://www.ryadel.com/en/fix-mysql-server-gone-away-packets-order-similar-mysql-related-errors/>
+
+### SQL注入在 PHP v5.2.6.上不正常
+
+PHP 5.x 的维护已于 2019 年 1 月终止，因此我们建议您使用当前 7.x 版本运行 DVWA，如果您一定要使用 5.x ……
+
+如果您使用的是 PHP v5.2.6 或更高版本，则需要执行以下操作才能使 SQL 注入和其他漏洞发挥作用。
+
+在`.htaccess`中：
+
+将以下设置：
+
+```php
+<IfModule mod_php5.c>
+    php_flag magic_quotes_gpc off
+    #php_flag allow_url_fopen on
+    #php_flag allow_url_include on
+</IfModule>
+```
+
+替换为：
+
+```php
+<IfModule mod_php5.c>
+    magic_quotes_gpc = Off
+    allow_url_fopen = On
+    allow_url_include = On
+</IfModule>
+```
+
+#### 命令行注入 不工作
```

Comment:
Nit: logically this should also be header level 3: `###` instead of `####`.

I will add this to the PR to fix the original English README.